scene/2d/box2d_joints.h

```diff
@@ -28,12 +29,20 @@
 
 class Box2DWorld;
 
+// TODO maybe relevant joints should inherit some LocalAnchorBearer class (name TBD)
+//     It would store local anchors and their accessors
+//     This would not be a Godot class
+//     Inheritance: Box2DJoint -> LocalAnchorBearer -> Box2D[Revolute, etc.]Joint
+// dunno, needs a lot more thought
+
```

Comment:
@jordo If you have time, do you have any thoughts on what I commented here?

Reply:
The goal would be to remove a lot of duplicate code

Reply:
for reference it says:
```
// TODO maybe relevant joints should inherit some LocalAnchorBearer class (name TBD)
//     It would store local anchors and their accessors
//     This would not be a Godot class
//     Inheritance: Box2DJoint -> LocalAnchorBearer -> Box2D[Revolute, etc.]Joint
// dunno, needs a lot more thought
```

Reply:
function binding would still be duplicated in all relevant joints, but at least it isn't duplicate logic

the reason for not registering it as a Godot class is so that it doesn't show up as an abstract node type like CanvasItem, Box2DJoint or Joint2D because that seems annoying